lib/protoflow-blocks/src/blocks/text.rs

```diff
@@ -2,29 +2,81 @@
 
 pub mod text {
     use super::{
-        prelude::{Cow, Named},
-        BlockConnections, BlockInstantiation,
+        prelude::{vec, Box, Cow, Named, Vec, String},
+        BlockConnections, BlockInstantiation, InputPortName, OutputPortName, System
     };
+    use protoflow_core::Block;
 
-    pub trait TextBlocks {}
+    pub trait TextBlocks {
+        fn concat_strings(&mut self) -> ConcatStrings;
+        fn concat_strings_by(&mut self, joiner: &str) -> ConcatStrings;
```

Comment:
Use "delimiter" everywhere instead of "joiner"

Reply:
done